### PR TITLE
implements DELETE /api/comments/:comment_id

### PR DESCRIPTION
**Pull request by steelfish1010**

### CODE REVIEW

File: models/comments.model.js [hunk 1]
```diff
@@ -22,3 +22,16 @@ exports.addCommentByArticleId = async (comment, article_id) => {
   );
   return rows[0];
 };
+
+exports.removeCommentById = async (comment_id) => {
+  if (!parseInt(comment_id)) {
+    return Promise.reject({ status: 400, msg: "comment_id is not a number" });
```
Comment (xmenbteam) [inline models/comments.model.js]: This is for sql to deal with!

File: models/comments.model.js [hunk 2]
```diff
@@ -22,3 +22,16 @@ exports.addCommentByArticleId = async (comment, article_id) => {
...
+  }
+  const { rows } = await db.query(`SELECT comment_id FROM comments;`);
+  const validCommentIds = rows.map((id) => id.comment_id.toString());
+  if (!validCommentIds.includes(comment_id)) {
+    return Promise.reject({ status: 404, msg: "comment_id does not exist" });
+  } else {
```
Comment (xmenbteam) [inline models/comments.model.js]: All of this is SQL stuff. Imagine Facebook pulling down every comment ever onto your local machine to check if a comment exists.

**Review by xmenbteam — CHANGES_REQUESTED** [review]
Great! Just be wary of when to use sql etc. - deleting comment functions are still subject to sql errors etc!

**Comment by steelfish1010** [comment]
Ah yes, I've just written a load of unnecessary code! Makes sense and I've updated it now

**Pull request merged**